### PR TITLE
Fix links in documentation

### PR DESCRIPTION
**Pull request by Nepoh**
Fixing some broken internal and external links in the documentation

### Your pull request should…

* [x] have a concise title
* [x] [close an accompanying issue](https://help.github.com/en/articles/closing-issues-using-keywords) if one exists
* [x] [be against the correct branch](https://docs.opencast.org/develop/developer/development-process#acceptance-criteria-for-patches-in-different-versions)
* [x] include migration scripts and documentation, if appropriate
* [x] pass automated tests
* [x] have a clean commit history
* [x] [have proper commit messages (title and body) for all commits](https://medium.com/@steveamaza/e028865e5791)

### CODE REVIEW

File: docs/guides/developer/docs/develop/admin-ui/style/icons.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ icons are based on [Font Awesome](http://fontawesome.io/) which gives us scalabl
 size, color, drop shadow, and anything that is provided by CSS.
 
 For more links and license information refer to [References - Font
-Awesome](/modules/admin-ui/style/references/#font-awesome).
+Awesome](references.md/#font-awesome).
```
Comment (ziegenberg) [inline docs/guides/developer/docs/develop/admin-ui/style/icons.md:12]: Are you sure this link will work?

**Review by ziegenberg — COMMENTED** [review]
Looks good to me, except for the one change I'm not sure about.

File: docs/guides/developer/docs/develop/admin-ui/style/icons.md (hunk 1 above)
Comment (Nepoh) [reply docs/guides/developer/docs/develop/admin-ui/style/icons.md:12]: I built the docs locally to check (mkdocs figures out those internal links between markdown files). Not 100% sure how the docs are built for the website tbh, but at least it seems to work in [index.md](https://github.com/opencast/opencast/blob/develop/docs/guides/developer/docs/develop/admin-ui/style/index.md?plain=1)

**Review by gregorydlogan — APPROVED** [review]
Checked the links, they work.

**Pull request merged**